aten/src/ATen/native/TensorConversions.cpp

```diff
@@ -1235,6 +1235,22 @@ Tensor sparse_compressed_to_sparse_csr(const Tensor& self, c10::optional<int64_t
       self.layout());
 }
 
+Tensor sparse_compressed_to_sparse_csc(const Tensor& self, c10::optional<int64_t> dense_dim_opt) {
+  if (dense_dim_opt.has_value()) {
+    AT_ERROR("sparse_compressed_to_sparse_csc conversion does not support specifying number of dense dimensions");
+  }
+  if (self.layout() == kSparseCsr) {
+    return sparse_compressed_to_flipped(self, c10::nullopt, "to_sparse_csc");
+  }
```

Comment:
nit: `to_sparse_csr`.

Reply:
`to_sparse_csc` seems correct as this function is about converting sparse compressed tensors to sparse CSC tensors.